inputFiles/poromechanics/PoroElastic_staircase_co2_3d_sequential.xml

```diff
@@ -0,0 +1,398 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+
+    <CompositionalMultiphaseReservoirPoromechanics
+      name="reservoirSystem"
+      solidSolverName="linearElasticity"
+      reservoirAndWellsSolverName="reservoirAndWells"
+      logLevel="1"
+      targetRegions="{ channel, barrier, wellRegion1, wellRegion2 }">
+      <NonlinearSolverParameters
+        couplingType="Sequential"
+        subcycling="1"
+        newtonTol="1.0e-8"
+        lineSearchAction="None"
```

Comment:
Is this disabled because it doesn't work or just not needed?

Reply:
just not needed and also my personal preference about line search

Reply:
sorry, clarification here, coupled solver currently does not support line search https://github.com/GEOS-DEV/GEOS/blob/5b20fd397c152fadc4e7f12b6209580221295634/src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp#L565